.pre-commit-config.yaml

```diff
@@ -10,8 +10,15 @@ exclude: >
   )
 repos:
   # Note: hooks that add content must run before ones which check formatting, lint, etc
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.4.0
+    hooks:
+    -   id: check-shebang-scripts-are-executable
+    -   id: check-yaml
+    -   id: end-of-file-fixer
+    -   id: trailing-whitespace
```

Comment:
We do have markdown files in the repo. We should add `args: [--markdown-linebreak-ext=md]` [like it suggests](https://github.com/pre-commit/pre-commit-hooks#trailing-whitespace) I think.

Reply:
Yeah, good call

Reply:
Updated. No `.md` files were harmed in the making of this PR.